mutator/loop/range_break.go

```diff
@@ -0,0 +1,79 @@
+package loop
+
+import (
+	"go/ast"
+	"go/token"
+	"go/types"
+
+	"github.com/avito-tech/go-mutesting/mutator"
+)
+
+func init() {
+	mutator.Register("loop/range_break", MutatorLoopRangeBreak)
+}
+
+func checkIsRangeStatement(node ast.Stmt) bool {
+	switch node.(type) {
+	case *ast.RangeStmt:
+		return true
+	}
+
+	return false
+}
+
+// MutatorLoopRangeBreak implements a mutator to add a break to range-loop body.
+func MutatorLoopRangeBreak(_ *types.Package, _ *types.Info, node ast.Node) []mutator.Mutation {
+	var list []ast.Stmt
+
+	switch n := node.(type) {
+	case *ast.BlockStmt:
+		list = n.List
+	case *ast.CaseClause:
+		list = n.Body
+	}
```

Comment:
if we will add deafult case we can make return asap

Reply:
ty, done